packages/flutter/lib/src/material/chip_theme.dart

```diff
@@ -299,6 +301,17 @@ class ChipThemeData extends Diagnosticable {
   /// The chip is selected when [selected] is true.
   final Color secondarySelectedColor;
 
+  /// Color of the chip's shadow when the elevation is greater than 0.
+  ///
+  /// If null, the chip defaults to [Colors.black].
+  final Color shadowColor;
+
+  /// Color of the chip's shadow when the elevation is greater than 0 and the
+  /// chip is selected.
+  ///
+  /// If null, the chip defaults to [Colors.black].
+  final Color selectedShadowColor;
```

Comment:
This comment should either weave in a reference to [shadowColor] or include a see also section.

Reply:
Done